.travis.yml

```diff
@@ -82,27 +76,38 @@ before_install:
   - echo $LC_ALL
   - $CXX -v
   - export PYTHONPATH=$PYTHONPATH:$INSTALL_DIR/lib/python/;
-  - if [ "$NRN_ENABLE_PYTHON_DYNAMIC" == "ON" ]; then
-      pyenv global 2.7.15 3.7.1;
-    else
-      pyenv global $PYTHON_VERSION;
-    fi
 install:
+  - $PIP install --user setuptools;
   - $PIP install --user scipy matplotlib bokeh ipython cython --upgrade;
+  # add cython location to PATH
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      export CYTHON_LIB=$($PIP show cython | grep Location);
+      export CYTHON_EXE=$(echo $CYTHON_LIB | awk '{print $2}' | awk -F "lib" '{print$1}')"bin";
+      export PATH=$CYTHON_EXE:$PATH;
+      echo $PATH;
+    fi
 
 #=============================================================================
 # Build, install and test
 #=============================================================================
 script:
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      export CONFIG_OPTIONS="$CONFIG_OPTIONS --with-readline=no";
```

Comment:
Without "--with-readline=no", I got the error msg during compilation: 
dyld: Symbol not found: _rl_event_hook
  Referenced from: /Users/travis/install/x86_64/lib/liboc.0.dylib
  Expected in: flat namespace
 in /Users/travis/install/x86_64/lib/liboc.0.dylib
/Users/travis/install/share/nrn/demo/neuron: line 13: 35169 Abort trap: 6           "${NRNIV}" -dll "/Users/travis/install/share/nrn/demo/release/x86_64/.libs/libnrnmech.so" "$@"

Reply:
this is expected